app/src/main/res/layout/item_repo.xml

```diff
@@ -1,67 +1,48 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-              android:layout_height="wrap_content"
-              android:orientation="vertical">
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
 
     <android.support.v7.widget.CardView
         android:id="@+id/card_view"
         style="@style/CardViewContainer"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        >
+        android:layout_height="wrap_content">
 
         <LinearLayout
             android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:orientation="vertical">
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:paddingBottom="4dp"
+            android:paddingLeft="8dp"
+            android:paddingRight="8dp"
+            android:paddingTop="4dp">
 
             <TextView
                 android:id="@+id/item_repo_name"
-                style="@style/ViewBuildText.TwoLines.Title.NoImage"
+                style="@style/ViewBuildText.TwoLines.Big"
                 android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                />
+                android:layout_height="wrap_content" />
 
-            <LinearLayout
+            <TextView
+                android:id="@+id/item_repo_duration"
+                style="@style/ViewBuildText"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:orientation="horizontal">
-
-                <ImageView
-                    style="@style/ViewBuildImageIcon"
-                    app:srcCompat="@drawable/ic_duration_grey600_16dp"
-                    />
+                android:drawableLeft="@drawable/ic_duration_grey600_16dp"
```

Comment:
1. The drawableLeft ensures the text and the icon are nicely aligned. If you carefully look at the image at the top you can see that the icon is a little bit off in the old version.
2. Much less views to inflate -> Faster performance
3. Less code -> Easier to read
4. If you want to change the alignment in the future again and it should not be possible with drawableLeft you can still go back to the ImageView TextView approach.